### PR TITLE
Update how to document ballerina code

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-document-ballerina-code.md [hunk 1]
```diff
@@ -125,13 +125,13 @@ First, let's create a new Ballerina project:
 $ mkdir myproject
 $ cd myproject
 $ ballerina init -i
-Create Ballerina.toml [yes/y, no/n]: (n) y
-Organization name: (nirmal) y
+Create Ballerina.toml [yes/y, no/n]: (y) y
+Organization name: (natasha) y
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-document-ballerina-code.md]: Can we put some sample value here?

File: website/two-column-pages/docs/learn/how-to-document-ballerina-code.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ A typical project structure of a Ballerina project is like this:
   .gitignore
   Ballerina.toml       # Configuration that defines project intent
   .ballerina/          # Internal cache management and contains project repository
-                       # Project repository is built or downloaded module dependencies
+                       # Project repository is built module dependencies
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-document-ballerina-code.md]: Does this mean project repository is built using module dependencies?

Comment (samgnaniah) [reply website/two-column-pages/docs/learn/how-to-document-ballerina-code.md]: "# Project repository is built module dependencies" doesn't give such a clear understanding though. Can we consider improving this at some point?

Comment (NatashaWso2) [reply website/two-column-pages/docs/learn/how-to-document-ballerina-code.md]: We can say "Project repository contains built module artifacts of the project". WDYT?

Comment (samgnaniah) [reply website/two-column-pages/docs/learn/how-to-document-ballerina-code.md]: +1.

File: website/two-column-pages/docs/learn/how-to-document-ballerina-code.md [hunk 3]
```diff
@@ -81,8 +81,9 @@ A typical project structure of a Ballerina project is like this:
 /
   .gitignore
   Ballerina.toml       # Configuration that defines project intent
-  .ballerina/          # Internal cache management and contains project repository
-                       # Project repository is built or downloaded module dependencies
+  .ballerina/          # Internal cache management and contains the project repository
+                       # Project repository contains compiled module binaries
+    module1.balo
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn/how-to-document-ballerina-code.md]: is this indented? If so, shouldn't this look consistent with the indented list below, in target section?

**Pull request merged**